### ISSUE TITLE
Plaintext Export does not contain MMS group messages

### ISSUE THREAD

**Issue by joeposner** [johanw666/Signal-Android#44]
Let's begin with a checklist: Replace the empty checkboxes [ ] below with checked ones [x] accordingly. -->

- [x ] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

I'm using your fork of Signal and have successfully imported a backup database from Signal; old group chant messages appear as expected in that import.

However, when I use the Plaintext export, these MMS messages do not appear to be in the exported file. A grep of the SignalPlaintextBackup.xml file for Signals standard group Id returns no values. Ex: 

`grep nal_mms_group SignalPlaintextBackup.xml`

Likewise, greps in that file for `mms` or text from a group chat produces no results. 

And, as expected with the results of those greps, an import with SMS Backup and Restore, the group chats do not appear

**Comment by johanw666** [comment]
Yes, media messages are not saved in the xml export. Adding the text part of those to the export is on my planning.

**Comment by joeposner** [comment]
Would you consider accepting a pull request from me to add this feature?

**Comment by johanw666** [comment]
Certainly, it would safe me some work. I have already made provisions in the code for media message export but it does not work yet and is behind an if (false). It is in PlaintextBackupExporter.java in exportPlaintext() and in some changes in mmsDatabase.java.
